src/transformers/models/auto/modeling_auto.py

```diff
@@ -411,6 +413,7 @@
         ("beit", "BeitForSemanticSegmentation"),
         ("data2vec-vision", "Data2VecVisionForSemanticSegmentation"),
         ("dpt", "DPTForSemanticSegmentation"),
+        ("fan", "FANForSemanticSegmentation"),
```

Comment:
Can you also add the proper entries in `feature_extraction_auto` and `image_processing_auto`?